ci-operator/step-registry/hypershift/azure/create-selfmanaged-guests/hypershift-azure-create-selfmanaged-guests-commands.sh

```
@@ -98,6 +98,29 @@ echo "$(date) Creating OAuth LB self-managed cluster: ${OAUTH_LB_NAME}"
   ${COMMON_FLAGS} &
 OAUTH_LB_PID=$!
 
+# Create HA upgrade cluster with N-1 release image (for upgrade + etcd chaos tests)
+UPGRADE_NAME="$(echo -n "${PROW_JOB_ID}-upg"|sha256sum|cut -c-20)"
+UPGRADE_RELEASE="${OCP_IMAGE_N1:-${RELEASE_IMAGE}}"
+echo "$(date) Creating HA upgrade cluster: ${UPGRADE_NAME} with image ${UPGRADE_RELEASE}"
+/usr/bin/hypershift create cluster azure \
+  --name="${UPGRADE_NAME}" \
+  --control-plane-availability-policy=HighlyAvailable \
+  --node-pool-replicas=${HYPERSHIFT_NODE_COUNT} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)